### DISCUSSION TITLE
Azure AD Auth: The request body must contain the following parameter: 'client_id'.

### DISCUSSION THREAD

**Discussion by sgl-mnrch** [requarks/wiki#6124] [Error / Bug Report]
When using the AzureAD Auth strategy, on the first try the user is presented this Error message by AzureA

`The request body must contain the following parameter: 'client_id'.` 

After reloading the page, everything works as expected.

I checked the relevant setting in the auth path, but everything is as documented here https://docs.requarks.io/auth/azure

Any hints?

**Comment by Te-k** [comment]
Random idea: it could be that you have "access tokens" enabled instead of "ID tokens" in your Azure app. Have you checked that?

**Comment by sgl-mnrch** [comment]
Yes, it is set on ID tokens.
